ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/vlan_interfaces.py

```diff
@@ -151,7 +151,7 @@ def _check_virtual_router_mac_address(vlan_interface_config: dict, variables: li
                 vlan_interface_config["ip_helpers"] = ip_helpers
 
         if get(svi, "ospf.enabled") is True and get(vrf, "ospf.enabled") is True:
-            vlan_interface_config["ospf_area"] = svi["ospf"].get("area", 0)
```

Comment:
There is no schema update for this change.

Reply:
Schema already set as a string, including default value:

```yaml
$defs:
  svi_settings:
    keys:
      ospf:
        keys:
          area:
            type: str
            convert_types:
              - int
            description: OSPF area ID.
            default: "0"
```